utils/keyframe/csv_creation.py

```diff
@@ -69,11 +70,23 @@ def milliseconds_to_time_string(milliseconds):
     return time_string
 
 
-def create_keyframes_csv(
+def create_csv(
     keyframes_csv_input_dir: str = "videos/keyframes",
     scenes_csv_input_file: str = "videos/video_scenes/scene_list.csv",
-    output_file: str = "videos/keyframes/extracted_keyframes.csv"
+    output_csv_filepath: str = "videos/keyframes/extracted_keyframes.csv"
```

Comment:
This is exactly what I was hoping we would not do. The structure I remember us agreeing on Monday would place the keyframes in a unique folder per video, the same as what is being done for the scenes, where we also dont have one single 'video_scenes' folder. 

Take this following example:
1. We download a video called 'test.mp4' → This video is downloaded to 'videos/test.mp4'
2. We extract the scenes from this video → The scene-files are placed in a directory called 'videos/test_scenes' and the csv files location is 'videos/test_scenes/scene_list.mp4'.
3. Following this structure, the logical next step is to extract the keyframes. If we do this consistently we would place the keyframes in a directory called 'videos/test_keyframes/'. The corresponding main csv would then be found under 'videos/test_keyframes/extracted_keyframes.csv'.

This way we have a simple way of placing each videos keyframes in unique non-colliding directories. The naming of this directory could be done directly in this function by calling split('_') on the scenes directory name and appending '_keyframes' to this string to get the keyframes name. No more passing parameters where they are not really needed. If we dont do this we will have to redefine the parameter for each new video, or call the code I have just outlined outside of this function which would unnecessarily blow out the amount of code needed in the pipeline.